ray-operator/Dockerfile

```diff
@@ -13,13 +13,14 @@ RUN go mod download
 COPY main.go main.go
 COPY api/ api/
 COPY controllers/ controllers/
+COPY rpc/ rpc/
```

Comment:
We may change how to organize the codes later. Let's concentrate on the functionality and let's adapt to the new changes later.